code/game/atoms_movable.dm

```diff
@@ -29,11 +29,10 @@
 	areaMaster = get_area_master(src)
 
 /atom/movable/attempt_init()
-	if(ticker && ticker.current_state >= GAME_STATE_SETTING_UP)
-		var/turf/T = get_turf(src)
-		if(T && space_manager.is_zlevel_dirty(T.z))
-			space_manager.postpone_init(T.z, src)
-			return
+	var/turf/T = get_turf(src)
+	if(T && SSatoms.initialized != INITIALIZATION_INSSATOMS && space_manager.is_zlevel_dirty(T.z))

```

Comment:
Part of me feels like using `INITIALIZATION_INSSATOMS` here is gonna bite me and future coders in the butt in the future